requirements.txt

```diff
@@ -1 +1 @@
-tableschema>=1.0.3,<1.20.4
+frictionless==5.18.0
```

Comment:
^ for minor or >= would be better

Reply:
`>=` has often resulted in situations where things mysteriously break because a patch was released...but we should be able to automatically allow patch releases.